### PR TITLE
[WT-33] Feat: add bridge service

### PR DESCRIPTION
**Pull request by JoanVicens**
Added a service to call to be able to call bridge endpoints

### CODE REVIEW

File: src/externals/bridge/bridge.service.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    };
+
+    Logger.log(
+      '[INXT removeFile]: User: %s, Bucket: %s, File: %s',
+      user.bridgeUser,
+      bucket,
+      bucketEntry,
+    );
+
+    await this.httpClient.delete(
```
Comment (fabioespinosa) [inline src/externals/bridge/bridge.service.ts:49]: You can remove async and await and just return the promise here

Comment (JoanVicens) [reply src/externals/bridge/bridge.service.ts:49]: The http client is returning a `Promise<AxiosResponse<any, any>>`

**Comment by JoanVicens** [comment]
> is it ready for review? or draft?

It is ready, was checking the `hashSha256` method, it was surrounded by a try catch in drive-server but I think its not necessary

File: src/externals/bridge/bridge.module.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import { BridgeService } from './bridge.service';
+
+@Module({
+  imports: [ConfigModule, HttpClientModule],
+  controllers: [],
+  providers: [
+    BridgeService,
+    {
+      provide: CryptoService,
+      useValue: new CryptoService(),
```
Comment (fabioespinosa) [inline src/externals/bridge/bridge.module.ts:14]: Why do you have to do this like this with CryptoService here?

Comment (JoanVicens) [reply src/externals/bridge/bridge.module.ts:14]: The `CryptoService` was not made "@Injectable()", instead is a singleton. I could use `CryptoService.getInstance()` too

**Pull request merged**